composer.json

```diff
@@ -4,6 +4,7 @@
     "description": "TYPO3 Flow package that integrates MailChimp into your Neos site / Flow Application",
     "license": "GPL-3.0+",
     "require": {
+        "typo3/neos": "^2.3",
```

Comment:
This package doesn't depend on Neos for a reason ;)
It can be used with Flow alone and I'd like to keep it that way. Any reason why you added this dependency?

Reply:
@bwaidelich: I think he might want to fix an issue that cannot load "TYPO3\Neos\Controller\Module\AbstractModuleController" namespace. I've installed this package version 3.0 in my flow application, and I got the above issue. Do you have any ideas?

Reply:
@cbunhok you're right, I wasn't aware that the compiler requires this class - and the same for the Form Finisher that requires `neos/form` to be installed. I see if I can work around this

Reply:
..There seems to be no easy fix, so 879b815 .. :-/